stores/stores.js

```diff
@@ -26,3 +31,25 @@ export const act_command = derived([devices, activeDevice, block0_31, block32_63
       "hf_block":{"b0":${$hf_block[0]}, "b1":${$hf_block[1]}, "b2":${$hf_block[2]}, "b3":${$hf_block[3]}},
       "lf_block":{"b0":${$lf_block[0]}, "b1":${$lf_block[1]}, "b2":${$lf_block[2]}, "b3":${$lf_block[3]}}
     } } }`);
+
+
+export const preset_display = readable([
+  { name: "flashall", 
+    array: [[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35]]},
```

Comment:
These should be programmatic and concise. Since we only have a single hexagon configuration, this works for now, but otherwise, this is a bad pattern.

Another rule of thumb, unnamed magic numbers are unacceptable. There should be an obvious trail back to the reason for every hardcoded number. The fewer there are, the fewer mistakes and the easier it is to make changes in the future.
```suggestion
    array: [[...Array(36).keys()]},
```

We can leave the rest of these for a later revision when we handle stitchable fabrics.